airflow/ui/src/router.tsx

```diff
@@ -26,8 +26,10 @@ import { Overview } from "src/pages/DagsList/Dag/Overview";
 import { Runs } from "src/pages/DagsList/Dag/Runs";
 import { Run } from "src/pages/DagsList/Run";
 import { Dashboard } from "src/pages/Dashboard";
-import { ErrorPage } from "src/pages/Error";
-import { Events } from "src/pages/Events";
+
+import { Tasks } from "./pages/DagsList/Dag/Tasks";
+import { ErrorPage } from "./pages/Error";
+import { Events } from "./pages/Events";
```

Comment:
```suggestion
import { Tasks } from "src/pages/DagsList/Dag/Tasks";
import { ErrorPage } from "src/pages/Error";
import { Events } from "src/pages/Events";
```

I might open a PR to add more aliases so that we can just call `pages/Events`

Reply:
Thanks, done. I guess I did this during rebase after runs PR merge.